manual/cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/CallTest.scala

```diff
@@ -74,28 +74,29 @@ class CallTest extends RefcardTest with QueryStatisticsTestSupport {
 ### assertion=labels
 //
 
-CALL db.labels
+CALL db.labels() YIELD label
 ###
 
-This invokes the built-in procedure db.labels, which lists all in-use labels in the database.
+This shows a standalone call to the built-in procedure +db.labels+ to list all labels used in the database.
+Note that required procedure arguments are given explicitly in brackets after the procedure name.
 
-### assertion=arg
+### assertion=arg parameters=arg
 //
 
-CALL procWithArg('foo')
+CALL java.stored.procedureWithArgs
 ###
 
-For a procedure that takes arguments, these can be provided explicitly.
+Standalone calls may omit +YIELD+ and also provide arguments implicitly via statement parameters, e.g. a standalone call requiring one argument `input` may be run by passing the parameter map `{input: 'foo'}`.
```

Comment:
There where three reasons for this AFAIR

(1) Two make standalone calls as small as possible in terms of bolt because they are our fast path for performance critical tasks
(2) Two allow passing arguments via parameters because that felt natural from a driver POV and also again makes the `CALL` smaller
(3) `YIELD` is mandatory inside a larger query because of the Cypher rule that all identifiers must be introduced explicitly; so standalone calls are really the short-cut exception